source/configure/plugins-configuration-settings.rst

```diff
@@ -406,6 +406,38 @@ The URL to a running `rtcd <https://github.com/mattermost/rtcd>`__ service insta
 
 This is an optional field. Changing this setting requires a plugin restart to take effect.
 
+Enable call recordings (beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+**True**: Allow call hosts to record meeting video and audio on the cloud. Recordings include the entire call window view along with participants' audio track and any shared screen video.
+
+**False**: (Default) Call recording functionality is not available to hosts.
+
+Changing this setting requires a plugin restart to take effect.
+
+Job service URL
+~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+The URL to the instance where call recordings will be stored.
```

Comment:
I think the description should be changed here to reflect what this is really about. The URL points to a running job service where all the processing related to recordings happens. The recordings files produced are ultimately stored in Mattermost, through the existing file attachment mechanism.